data/gui/widget/progress_bar_default.cfg

```diff
@@ -56,3 +56,56 @@
 	[/resolution]
 
 [/progress_bar_definition]
+
+[progress_bar_definition]
```

Comment:
Should this go in a separate file? I think the usual thing is one definition per file, though there are probably other exceptions to that too…

Reply:
Updated.